VideoWeb/VideoWeb/ClientApp/src/app/shared/chat-input-box/chat-input-box.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="chat-input-group">
     <div [ngClass]="isInputInvalid ? '' : 'govuk-form-group  govuk-!-margin-bottom-2'"
          class="chat-input-text-button-group govuk-!-margin-left-2">
+         <label for="new-message-box"></label>
```

Comment:
There is no content for this label. We need to have an English and a Welsh content.

Reply:
if we put some English and Welsh to the label means we need hide this element